godal.go

```diff
@@ -437,6 +437,74 @@ func (band Band) Histogram(opts ...HistogramOption) (Histogram, error) {
 	return h, nil
 }
 
+//Statistics returns if present and flag as true.
+//If approximated statistics already computed, approximated statistics are returned.
+//Return false and error if no statistics are availables.
+func (band Band) GetStatistics(opts ...StatisticsOption) (bool, Statistics, error) {
+	sopt := statisticsOpts{}
+	for _, s := range opts {
+		s.setStatisticsOpt(&sopt)
+	}
+	var min, max, mean, std C.double
+	cgc := createCGOContext(nil, sopt.errorHandler)
+	ret := C.godalGetRasterStatistics(cgc.cPointer(), band.handle(),
+		(C.int)(sopt.approx), &min, &max, &mean, &std)
+	if err := cgc.close(); err != nil {
+		return false, Statistics{}, err
+	}
+	if ret == 0 {
+		return false, Statistics{}, nil
+	}
+	var ap bool = sopt.approx != 0
+	s := Statistics{
+		Approximate: ap,
+		Min:         float64(min),
+		Max:         float64(max),
+		Mean:        float64(mean),
+		Std:         float64(std),
+	}
+	return true, s, nil
+}
+
+//Statistics returns from exact computation or approximation.
```

Comment:
comment/doc needs fixing (must start with `ComputeStatistics`) and a bit more information as to which options can be used, and specifying that a full scan might be necessary